components/script/body.rs

```diff
@@ -27,14 +33,16 @@ pub enum BodyType {
     Blob,
     FormData,
     Json,
-    Text
+    Text,
+    ArrayBuffer
 }
 
 pub enum FetchedData {
     Text(String),
     Json(JSValue),
     BlobData(DomRoot<Blob>),
     FormData(DomRoot<FormData>),
+    ArrayBuffer(RootedTraceableBox<Heap<JSVal>>),
```

Comment:
I'm bit unsure if I understood https://github.com/servo/servo/pull/20406#discussion_r176900532 fully around `use it for the Json variant as well.` - would you mind elaborate bit more?

Reply:
I mean that the existing `Json(JSValue)` is similarly GC-unsafe, and it should store a `RootedTraceableBox<Heap<JSVal>>` instead.